jsonnet/kube-prometheus/versions.json

```
@@ -9,5 +9,6 @@
   "prometheusOperator": "0.79.2",
   "kubeRbacProxy": "0.18.2",
   "configmapReload": "0.14.0",
-  "pyrra": "0.6.4"
+  "pyrra": "0.6.4",
+  "metricsServer": "0.6.4"
```

Lets use latest release

```suggestion
  "metricsServer": "0.7.2"
```

---

this has been resolved!